img-src/Dockerfile

```diff
@@ -1,16 +1,10 @@
-FROM debian:buster
+FROM alpine
 
-RUN apt update
-RUN apt upgrade -y
-RUN DEBIAN_FRONTEND=noninteractive apt install -y \
-    bash \
+RUN apk add -U \
     iptables
```

Comment:
To make the plugin work, we need both iptables and nftables.
Moreover, due to the switch [here](https://github.com/KatharaFramework/NetworkPlugin/blob/master/go-src/src/common_utils.go#L47) and [here](https://github.com/KatharaFramework/NetworkPlugin/blob/c0bc61a5ad16de974c963e4f33e559e1dc51232a/go-src/src/common_utils.go#L51) we need also `dpkg` installed into the container (`update-alternatives` is a part of the `dpkg` package)
So this should install all this requirements and be something like this
`apk add -U iptables dpkg nftables`